core/server/api/v2/users.js

```diff
@@ -126,7 +126,11 @@ module.exports = {
                 return Promise.all([
                     models.Accesstoken.destroyByUser(frame.options),
                     models.Refreshtoken.destroyByUser(frame.options),
-                    models.Post.destroyByAuthor(frame.options)
+                    models.Post.destroyByAuthor(frame.options),
+                    frame.options.transacting('roles_users')
```

Comment:
Thanks for your PR 👍 This works, but this is not really a bookshelf/ORM approach.

Do you mind if i push a commit to this PR?
Or let me know if you want to change your code, then i can help/advice you.

**Background: (in case you are curious):**
The user model should destroy it's relation automatically. Otherwise, any unit who calls `User.destroy` needs to handle removing the relations.